vlib/v/checker/tests/immutable_deref.vv

```diff
@@ -1,12 +0,0 @@
-struct Context {}
-
-const ctx_ptr = &Context(unsafe { nil })
-
-fn main() {
-	// TODO unsafe bug, having this declaration inside `unsafe` results in an error
-	x := &ctx_ptr
-	unsafe {
-		*x = &Context{}
-		_ = x
-	}
-}
```

Comment:
Restore this .vv/.out pair. Its purpose is to ensure, that the checker will not regress, and is not related to term.ui .

Reply:
I've deleted this test as it seemed to me that it abuses a behavior that should not work, but I can add it again

Reply:
Yes, please restore it.